crane/hooks/slack.py

```diff
@@ -267,7 +273,7 @@ def is_active(self):
                 fg="yellow",
             )
 
-        return settings.get("slack_token") and settings.get("slack_channel")
+        return self.token and self.slack_channels
```

Comment:
Is there a reason why there was settings.get('slack_token') instead of self.slack_token as we set these variables in init?

Reply:
I don't think so 🙊